webui.py

```diff
@@ -1959,9 +1965,9 @@ def run_RealESRGAN(image, model_name: str):
 
 txt2img_defaults = {
     'prompt': '',
-    'ddim_steps': 50,
+    'ddim_steps': 25,
     'toggles': [1, 2, 3],
-    'sampler_name': 'k_lms',
+    'sampler_name': 'k_euler_a',

```

Comment:
Do not add random changes that are not documented in the description of your PR.
Did you perhaps commit your own modifications by accident?